docs/platform/porting/graphics-and-ui.md

```diff
@@ -416,7 +416,7 @@ The output backend interface is mandatory. For more information, see [tdm_backen
 | `output_create_capture()`     | Creates a capture object of an output object. The backend module does not need to implement this function if the hardware does not have a capture device. | No  |
 | `output_set_status_handler()` | Sets an output connection status handler. The backend module must call the output status handler when the output connection status has been changed to let the TDM frontend know of the change. | No  |
 | `output_set_dpms_handler()`   | Sets an output DPMS handler. The backend module must call the output DPMS handler when the output DPMS has been changed to let the TDM frontend know of the change. | No  |
-| `output_get_hwc()`            | Gets a HWC object of an output object. The backend module returns the HWC object when the output has TDM_OUTPUT_CAPABILITY_HWC. | Yes  |
+| `output_get_hwc()`            | Gets an hwc object of an output object. The backend module returns the HWC object when the output has TDM_OUTPUT_CAPABILITY_HWC. | Yes  |
```

Comment:
Doubt:
In the first part of the sentence
"HWC object...." is changed to "hwc object"
In the second line, "HWC object" is not changed. 

Please verify.

Reply:
good point, I will change them.

Reply:
Thank you for updating the same.